lib/crawlers/company_crawler/company_list_crawler.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'company_crawler'
+require 'pathname'
+require 'fileutils'
+require 'csv'
+
+module Crawlers
+  class CompanyListCrawler
+    ROOT = Pathname.new(__FILE__).parent.parent.parent.parent
+    OUTPUT_PATH = Pathname.new("crawl_list_output.csv")
+    # Allows for appending a row to an existing output file so as not to overwrite
+    # This also means that if you recrawl a company, it'll have more than one entry in the output file
```

Comment:
Can we output this to a different file perhaps? Not an ideal solution to have more than one entry

Reply:
Should I just append the current timestamp to the file name when it's run?
i.e.
```ruby
OUTPUT_PATH = File.join(Rails.root, "crawl_list_output_#{Process.clock_gettime(Process::CLOCK_MONOTONIC).to_i}.csv")
```

Reply:
Also thinking it would be better to save the file to 'storage/csv'